lti_consumer/templates/html/student.html

```diff
@@ -0,0 +1,91 @@
+<%!
+import json
+from django.utils.translation import ugettext as _
+%>
+
+<h2 class="problem-header">
+    ## Translators:  "External resource" means that this learning module is hosted on a platform external to the edX LMS
+    ${display_name} (${_('External resource')})
+</h2>
+
+% if has_score and weight:
+    <div class="problem-progress">
+        % if module_score is not None:
+            ## Translators: "points" is the student's achieved score on this LTI unit, and "total_points" is the maximum number of points achievable.
+            (${_("{points} / {total_points} points").format(points=module_score, total_points=weight)})
+        % else:
+            ## Translators: "total_points" is the maximum number of points achievable on this LTI unit
+            (${_("{total_points} points possible").format(total_points=weight)})
+        % endif
+    </div>
+% endif
+
+<div
+    id="${element_id}"
+    class="${element_class}"
+    data-ask-to-send-username="${ask_to_send_username}"
+    data-ask-to-send-email="${ask_to_send_email}"
+>
+
+% if launch_url and launch_url !=  'http://www.example.com' and not hide_launch:
```

Comment:
was this logic particularly around launch_url != 'http://www.example.com' part of the xModule implementation? I'm guessing that URL is the field default. If so, I think it might be cleaner to scrub that in the python side, e.g. perhaps set launch_url to None and then keep this Mako conditional just be %if launch_url: